data/json/vehicleparts/vehicle_parts.json

```
@@ -1791,7 +1791,7 @@
     "name": { "str": "collapsible solar array" },
     "color": "light_blue",
     "broken_color": "light_gray",
-    "proportional": { "epower": 0.4, "durability": 0.5 },
+    "proportional": { "durability": 0.5 },
```

```suggestion
    "proportional": { "epower": 0.68, "durability": 0.5 },
```
Quick google shows collapsible panels average efficiency of 7%-15%.  Non foldable 15-22%.  I went ahead and improved the efficiency difference to 15/22.  68%.